parts/k8s/kubernetescustomscript.sh

```diff
@@ -3,14 +3,25 @@
 # This script runs on every Kubernetes VM
 # Exit codes represent the following:
 # | exit code number | meaning |
-# | 20 | Timeout waiting for docker install to finish |
 # | 3 | Service could not be enabled by systemctl |
 # | 4 | Service could not be started by systemctl |
 # | 5 | Timeout waiting for cloud-init runcmd to complete |
 # | 6 | Timeout waiting for a file |
+# | 7 | Error placing apt-mark hold on walinuxagent
```

Comment:
Added a bunch of new codes

Reply:
Are those CSE exit codes? It looks like they are cloud init exit codes. If so they should be separated from the CSE exit codes I think.

Reply:
Some of them are used only in cloud-init currently, kept them documented in here. If possible, it would make sense to have a common set of "provisioning" exit codes that cloud-init and CSE (and whatever) adhered to, so <provisioning context> + <exit code> gives us the whole picture. (Seems easier than keeping separate dictionaries per-provisioning context.)